src/js/providers/html5.js

```diff
@@ -392,6 +392,10 @@ define([
             if (_isMobile) {
                 // results in html5.controller calling video.play()
                 _sendBufferFull();
+                // If we're still paused, then the tag isn't loading yet due to mobile interaction restrictions.
+                if(!_videotag.paused){
+                    _this.setState(states.LOADING);
+                }
```

Comment:
Can you put this before `_sendBufferFull`? It would be a shame to set the state to loading after playback is stated.